docs/guides/Integration-Guides/data-privacy/changes-and-limitations-pii-data-architecture.md

```diff
@@ -15,6 +15,12 @@ In this guide, you can learn about the changes and limitations you must be aware
 
 ## Adaptation requirements for feature implementation
 
+In this section, you can learn what VTEX features are available but require adaptation for accounts using the PII data architecture.
+
+### Promotions - Customer cluster
+
+To use cluster promotions in a way that is compatible with the PII data architecture, you must use the `piiClusterExpressions` field to configure your promotions instead of `clusterExpressions`.
```

Comment:
```suggestion
To set up promotions for specific customer clusters in a way that is compatible with the PII data architecture, you must use the `piiClusterExpressions` field instead of `clusterExpressions` in [Promotions and Taxes API](https://developers.vtex.com/docs/api-reference/promotions-and-taxes-api#post-/api/rnb/pvt/calculatorconfiguration).
```